lib/matplotlib/text.py

```diff
@@ -309,6 +315,27 @@ def get_rotation_mode(self):
         """Return the text rotation mode."""
         return self._rotation_mode
 
+    def set_antialiased(self, b):
+        """
+        Set whether to use antialiased rendering.
+
+        Parameters
+        ----------
+        b : bool
+
+        Notes
+        -----
+        Antialiasing will be determined by ``rcParams['text.antialiased']``
+        and parameter ``antialiased`` will have no effect if the text contains
```

Comment:
```suggestion
        and parameter *antialiased* will have no effect if the text contains
```
or `*b*` for the current choice of parameter name.